lib/cmd/fh3/credential/delete.js

```diff
@@ -0,0 +1,28 @@
+/* globals i18n */
+module.exports = {
+  'desc' : i18n._('Delete credential bundle'),
+  'examples' :
+  [{
+    cmd : 'fhc credential delete --id=<bundle-id>',
+    desc : "Delete the credential bundle with the id=<bundle-id>"
+  }],
+  'demand' : ['id'],
+  'alias' : {
+    'id': 'i',
+    0: 'id'
+  },
+  'describe' : {
+    'id' : i18n._("Unique 24 character BundleID of the credential bundle that you want to delete"),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComma)